src/Sylius/Bundle/ResourceBundle/.gitignore

```diff
@@ -3,3 +3,6 @@ bin/
 
 composer.phar
 composer.lock
+
+tests/app/cache
```

Comment:
Why don't we use a temporary directory just like in `ThemeBundle/Tests/Functional/WebTestCase::getTmpDirPath`?